script/Arcueid/Day 6/06_00_ARC06_1B.txt

```diff
@@ -112,7 +112,8 @@ Straight ahead, on the other side of this door, the head of the Tohno family, Ak
 [sha:6befcc566f72ee05c1afe5bd29335440adb0d652]{
 -- Page 47, Offset 14036.
 -- ひどい魔窟だな、ここ。
-This place really is the cruel home of a devil.
+// to be rechecked
+This place really is the den of a devil.
```

Comment:
Might go with hell on this one, and restructure sentence a bit
> I may as well be entering hell itself.

![2022-01-22-211644_583x345_scrot](https://user-images.githubusercontent.com/2476799/150662015-397d9f3f-1f84-4b9c-88da-fd9ce7cfcea9.png)